src/Analysis/GWASV2/Components/AttritionTableWrapper/AttritionTable/AttritionTableRow.jsx

```diff
@@ -128,9 +132,16 @@ const AttritionTableRow = ({
     return value;
   };
 
+  const rowTypeDescription = (rowType) => {
+    if (rowType === 'Outcome') {
+      return 'Outcome Phenotype';
+    }
+    return rowType;
+  };
+
   return (
     <tr>
-      <td className='gwasv2-attrition-table--leftpad'>{rowType}</td>
+      <td className='gwasv2-attrition-table--leftpad'>{rowTypeDescription(rowType)}</td>
```

Comment:
{ `rowType` && `rowType` === 'Outcome' ? 'Outcome Phenotype' : `rowType` }

Reply:
sure, I can use that.

Reply:
done.